### ISSUE TITLE
Broken support for wasd.tv

### ISSUE THREAD

**Issue by mesb1** [yt-dlp/yt-dlp#4595]
### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2022.07.18** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Russia

### Provide a description that is worded well enough to be understood

Hi.

Few month ago, twitch analogue named wasd.tv was added
https://github.com/yt-dlp/yt-dlp/issues/3041

But it looks like design is slightly changed now and yt-dlp is not working properly.
Please take a look is it possible to fix it?

https://wasd.tv/kadavr?record=1086347
https://wasd.tv/StasSatori?record=919135
https://wasd.tv/spacemita/videos
https://wasd.tv/spacemita/videos?record=907755
https://wasd.tv/spacemita/clips
https://wasd.tv/spacemita/clips?clip=26804

For videos and clips it shows formats, but failed to load now.
For direc records it failed at all.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
c:\tmp>yt-dlp.exe -vUF https://wasd.tv/kadavr?record=1086347
[debug] Command-line config: ['-vUF', 'https://wasd.tv/kadavr?record=1086347']
[debug] Encodings: locale cp1251, fs utf-8, pref cp1251, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.07.18 [135f05e] (win32_exe)
[debug] Python 3.8.10 (CPython 64bit) - Windows-10-10.0.19044-SP0
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: avconv -bsfs
[debug] Checking exe version: ffprobe -bsfs
[debug] Checking exe version: avprobe -bsfs
[debug] exe versions: none
[debug] Optional libraries: Cryptodome-3.15.0, brotli-1.0.9, certifi-2022.06.15, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: 2022.07.18, Current version: 2022.07.18
yt-dlp is up to date (2022.07.18)
[debug] [generic] Extracting URL: https://wasd.tv/kadavr?record=1086347
[generic] kadavr?record=1086347: Downloading webpage
WARNING: [generic] Falling back on generic information extractor.
[generic] kadavr?record=1086347: Extracting information
[debug] Looking for video embeds
ERROR: Unsupported URL: https://wasd.tv/kadavr?record=1086347
Traceback (most recent call last):
  File "yt_dlp\YoutubeDL.py", line 1431, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1506, in __extract_info
  File "yt_dlp\extractor\common.py", line 644, in extract
  File "yt_dlp\extractor\generic.py", line 4108, in _real_extract
yt_dlp.utils.UnsupportedError: Unsupported URL: https://wasd.tv/kadavr?record=1086347
```

**Comment by pukkandan** [comment]
Workaround: use `https://wasd.tv/kadavr/videos?record=1086347`

**Comment by pukkandan** [comment]
Fixed by 22b22b7d5c9dafa1d3f2dac25522bdd8b4091de4
